src/librbd/io/ObjectRequest.h

```diff
@@ -173,6 +181,7 @@ class AbstractObjectWriteRequest : public ObjectRequest<ImageCtxT> {
   void send() override;
 
 protected:
+  uint64_t m_object_off, m_object_len;
```

Comment:
Nit: (while changing things)
```suggestion
  uint64_t m_object_off;
  uint64_t m_object_len;
```